base_layer/contacts/tests/contacts_service.rs

```diff
@@ -96,6 +96,10 @@ pub fn setup_contacts_service<T: ContactsBackend + 'static>(
         rpc_max_simultaneous_sessions: 0,
         rpc_max_sessions_per_peer: 0,
         listener_self_liveness_check_interval: None,
+        excluded_dial_addresses: vec![
+            "/ip4/172.2.3.4/tcp/18188".parse::<Multiaddr>().expect("will not fail"),
+            "/ip4/172.2.3.4/tcp/18189".parse::<Multiaddr>().expect("will not fail"),
```

Comment:
We should not do this, we should use `0.0.0.0` and exclude them. 
These are valid addresses and someone might actually have them and try and use them .
https://ipinfo.io/AS7018/172.0.0.0/12-172.2.0.0/22
https://www.techtarget.com/searchnetworking/answer/What-is-the-IP-address-0000-used-for#:~:text=On%20servers.-,IP%20address%200.0.,On%20PCs%20and%20client%20devices.